go.mod

```diff
@@ -226,7 +226,8 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/celo-org/op-geth v0.0.0-20240612131021-96a2a76ceaaa
+// replace github.com/ethereum/go-ethereum => github.com/celo-org/op-geth v0.0.0-20240612131021-96a2a76ceaaa
+replace github.com/ethereum/go-ethereum => github.com/celo-org/op-geth v0.0.0-20240612164035-ef9b9a19f53e
```

Comment:
Which branch does this reference? I don't see the commit in any branch.

Reply:
It's this branch @piersy put together https://github.com/celo-org/op-geth/tree/piersy/for-use-with-migrated-celo-datadir-use-gas-limit-differentiation-rebased-celo6

I believe he's planning to break it up into smaller PRs before merging into celo6, the first of which is this one https://github.com/celo-org/op-geth/pull/143 

So, if those changes aren't merged soon I'll just comment out this line in favor of the original one and add instructions to the README for now

Reply:
It depends, @piersy how long do you think it will be / what do you think makes most sense?

Reply:
I would just wait (header and block handling pr has been merged and tx handling pr is up), if this gets committed before those changes are on master then that commit will be broken at some point. Since either it will point to a commit on a branch that will get deleted, and so at that point you won't be able to build, or you can point it to celo6, but then the migration script won't work.

Reply:
Ok this (eb460e410416a7a66910f4cc271569710b979dc2) is the tip of celo6 and should be a good branch to use to support this script.

Reply:
```suggestion
replace github.com/ethereum/go-ethereum => github.com/celo-org/op-geth v0.0.0-20240621161032-eb460e410416
```